CONTRIBUTING.md

```diff
@@ -11,21 +11,40 @@ We actively welcome your pull requests.
 6. If you haven't already, complete the Contributor License Agreement ("CLA").
 
 ### Forking, cloning and running Rapid
-
 - Make sure to use the latest `node` with something like  `nvm install node`
 - Run `npm run all`
 - Run `npm run quickstart` to start the server
 
 ### Debugging and testing
-
 - In VSCode go to Run>Add Configuration... and select Launch Chrome
 - Examples of breakpoints
 - Conditional breakpoints
 - Some interesting files to break (init, etc)
 
+#### Code walkthrough
+
+Top-level Rapid Directories: 
+`css`: pretty self-explanatory, just contains our .css for styling things like the sidebars, top bars, buttons, etc. Side note: there is no .css 'inside' the map istelf- that's all styled using webGL. 
+`data`: static data files that define things like the imagery, data formats, language resources, and walkthrough data. 
+`dist`: the folder that all the built artifacts get served from. 
+`docs`: This folder gets filled whenever you run `npm run doc`. ( <@637767012415963173> -> For your doc writing endeavors!)
+`modules`: Where most of the real code lives. 
+`svg`: where all the svg icons live- anytime you see an icon in the map, or in the sidebars/UI- it comes from here!
```

Comment:
### SVG

The 'svg' directory serves as the repository for all SVG icons. Every time you encounter an icon within the map or in the sidebars/UI, it is sourced from this directory.